Demo/Components/LoanCalculator/LoanCalculatorDemoView.swift

```diff
@@ -159,6 +199,24 @@ extension LoanCalculatorDemoViewModel {
     }
 }
 
+private extension UIColor {
+    static func dynamicColor(defaultColor: UIColor, darkModeColor: UIColor) -> UIColor {
```

Comment:
Don't we have `dynamicColorIfAvailable` already?

Reply:
it's not public. this is part of the demo, I was unsure if to make the `dynamicColorIfAvailable` public as the internals depend on finniversKit as well the interface style support for iOS 13

Reply:
Right, but I think it's still useful to have this kind of extension in `FinniversKit`. I know it's a bit confusing to have two functions with similar names, but this `dynamicColor(defaultColor:darkModeColor)` could handle only the interface style support and `dynamicColorIfAvailable` would also check `FinniversKit.userInterfaceStyleSupport` stuff in addition to that. What do you think?

Reply:
I agree, I also saw myself needing other color related stuff, like the `UIColor(hex:)` initializer

Reply:
I will make a new PR exposing these stuff in FinniversKit